audio_transcription/whisperx/whisperx_model.py

```diff
@@ -1,6 +1,9 @@
 import sieve
 from typing import List
 from pydantic import BaseModel
+import os
+
+file_dir = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Code change here to make sure the readme can be read regardless of where the deploy command is run.